### PR TITLE
Stop celery tsunami

### PR DESCRIPTION
**Pull request by eviljeff**
* flattens chain of tasks into a single task
* removes pointless return values
* doesn't try to get a l10n en-US because we already got it from m-c (and it doesn't exist anyway)

### CODE REVIEW

File: src/olympia/files/management/commands/update_permissions_from_mc.py [hunk 1]
```diff
@@ -24,10 +22,9 @@ def handle(self, *args, **options):
 
         central_url = settings.WEBEXT_PERM_DESCRIPTIONS_URL
         locales_url = settings.WEBEXT_PERM_DESCRIPTIONS_LOCALISED_URL
+        amo_locales = [l for l in settings.AMO_LANGUAGES if l != 'en-US']
```
Comment (diox) [inline src/olympia/files/management/commands/update_permissions_from_mc.py]: Can you also exclude `dbg` ?

Comment (eviljeff) [inline src/olympia/files/management/commands/update_permissions_from_mc.py]: is that not a real locale?

Comment (diox) [inline src/olympia/files/management/commands/update_permissions_from_mc.py]: It's the debug locale :) https://addons-dev.allizom.org/dbg/firefox/

It should not be enabled in prod, but it'll make noise on dev.

Comment (eviljeff) [inline src/olympia/files/management/commands/update_permissions_from_mc.py]: `dbg` isn't in `AMO_LANGUAGES`

Comment (eviljeff) [inline src/olympia/files/management/commands/update_permissions_from_mc.py]: oh, I was just looking in settings_base

Comment (eviljeff) [inline src/olympia/files/management/commands/update_permissions_from_mc.py]: https://github.com/mozilla/addons-server/pull/4927/commits/9a7122e00b09f6a8a0aa2b1baf7188f55d6a1989

File: src/olympia/files/tasks.py [hunk 2]
```diff
@@ -44,6 +44,16 @@ def extract_webext_permissions(ids, **kw):
...
 @write
+def update_webext_descriptions_all(primary, additional, **kw):
+    """primary is a (url, locale) tuple; additional is a list of tuples."""
+    url, locale = primary
+    update_webext_descriptions(url, locale)
+    for url, locale in additional:
+        update_webext_descriptions(url, locale, create=False)
+
+
+@task
```
Comment (diox) [inline src/olympia/files/tasks.py]: No longer needs to be a task

**Pull request merged**